python/requirements_compiled.txt

```diff
@@ -1851,22 +1842,18 @@ qpd==0.4.4
     # via fugue
 querystring-parser==1.2.4
     # via mlflow
-    # via
-    #   raydp
-    #   tune-sklearn
-raydp==1.7.0b20231020.dev0
-    # via -r /home/ubuntu/ray/ci/../python/requirements/ml/data-requirements.txt
```

Comment:
raydp is gone

not sure if data wants to add this into test dependencies or not.

Reply:
yes still want it for test dependency.